lib/tilt/commonmarker.rb

```diff
@@ -6,8 +6,12 @@ class CommonMarkerTemplate < Template
     self.default_mime_type = 'text/html'
 
     OPTION_ALIAS = {
-      :smartypants => :SMART
+      :smartypants => :SMART,
+      :escape_html => :UNSAFE,
     }
+    INVERT_OPTIONS = [
+      :escape_html,
+    ]
```

Comment:
Unless you expect additional inverted options, I would avoid using a constant for this single value.  However, that change can be done later.